### PR TITLE
add seed file with new items, item categories, with some retired items.

### PR DESCRIPTION
**Pull request by cmacaulay**
👍 
- New seed file with:
-- Categories: "Wonder Witch", "Sweets", "Defence Objects", "Muggle Magic"
-- New items in these categories 
-- Some retired items 
- Adds paperclip logic to cart show view so that images added by admin load 

Notes: 
- Not sure if the paperclip logic needs to be added anywhere else 
- Should be merged after Courtney's - contains a lot of her changes

### CODE REVIEW

File: app/views/carts/show.html.erb [hunk 1]
```diff
@@ -25,7 +25,12 @@
       <tbody>
         <% @cart.items.each do |item, quantity| %>
         <tr>
-          <td class="avatar"><img src="<%=item.image_tag%>" width=75px height=75px class="circle"/></td>
+          <td class="avatar">
+            <% if item.image_tag %>
```
Comment (meyerhoferc) [inline app/views/carts/show.html.erb:29]: 👍

File: db/seeds.rb [hunk 2]
```diff
@@ -3,17 +3,22 @@
 require_relative '../app/models/item'
 
 def import_product_csv
-  CSV.foreach('db/csv/first_seed.csv', headers: true, header_converters: :symbol) do |row|
+  CSV.foreach('db/csv/all_wheezes.csv', headers: true, header_converters: :symbol) do |row|
```
Comment (meyerhoferc) [inline db/seeds.rb:6]: yay more seeds!

**Review by meyerhoferc — APPROVED** [review]
awesome, love the picture and excited for more seeds!
ready for second review & merge @valerietrudell @CjMoore

**Review by CjMoore — APPROVED** [review]
looks good. Love all the dumbledore images. merging now.

**Pull request merged**